client/homebrew/pages/vaultPage/vaultPage.jsx

```diff
@@ -248,6 +263,43 @@ const VaultPage = (props)=>{
 		</div>
 	);
 
+	const renderSortOption = ( optionTitle, optionValue)=>{
```

Comment:
Check tabs vs spaces on this file

Reply:
In my machine, there are no spaces, all tabs, not sure if you have the latest changes or if there is a display difference.